src/test/resources/application-test.yaml

```diff
@@ -45,9 +45,6 @@ spring:
   messages:
     encoding: ISO-8859-1
     basename: messages, messages-form-flow
-  jpa:
-    hibernate:
-      ddl-auto: create-drop
```

Comment:
Does this mean our DB data stays around for the next run?  I thought this aided in clearing out the db at the end of tests, but I could be wrong.

Reply:
The clearing out is handled by `org.springframework.boot.autoconfigure.flyway.FlywayMigrationStrategy`.

Reply:
It is actually https://github.com/codeforamerica/form-flow/blob/9c189f9ba6a22a66a03496eab42697c7063fc0c6/src/test/java/formflow/library/config/FlywayConfiguration.java#L8